pymatgen/io/lobster/outputs.py

```diff
@@ -387,14 +388,14 @@ def __init__(
             if len(data) == 0:
                 raise RuntimeError("ICOHPLIST file contains no data.")
 
-            # Which Lobster version?
+            # Determine LOBSTER version
             if len(data[0].split()) == 8:
                 version = "3.1.1"
             elif len(data[0].split()) == 6:
```

Comment:
I would first check if it holds true for the latest LOBSTER versions. I remember some back and forth in the output format 🙈

Reply:
Thanks! It currently gives `version` either being either `2.2.1` or `3.1.1` which is a bit misleading (it misled me in the first place, I thought this would only work for these two versions, until I read further): https://github.com/materialsproject/pymatgen/blob/b19508e0044d7fcbbe2102483998aa330680334c/pymatgen/io/lobster/outputs.py#L390-L397

Reply:
Yeah, sure is but afterwards, the ICOHP hasn't beem changed. We can call it differently but we need to figure this out for file passing

Reply:
Maybe we can also consider that only backwards compatibility for lobster v3 and v4 is needed? I'm not sure who would use that old lobster versions nowadays 🤔

Reply:
We still have these old test files and it works. Why not keeping it.

Reply:
> We can call it differently but we need to figure this out for file passing

Yes I just want to call it in a more intuitive manner instead of removing support for LOBSTER v2. And I believe you're the right person to suggest a new name :) Thanks in advance.

Reply:
out_file_type ?

Reply:
My bad, I mean the version, instead of `2.2.2` or `3.1.1`, perhaps we need some more descriptive values